controllers/quizController.js

```diff
@@ -0,0 +1,59 @@
+import * as QuizModel from '../models/QuizModel';
+import AppError from '../errors/AppError';
+
+const getQuizzes = async (req, res, next) => {
+  try {
+    const quizzes = await QuizModel.getAllQuizzes();
+    res.status(200).send({
+      payload: { quizzes }
```

Comment:
please return empty array if no quizzes were found

Reply:
model.find() already returns empty array by default if nothing found.

Reply:
Ok. Did not know that